### DISCUSSION TITLE
Установка Node.js ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#39] [Announcements]
# Установка Node.js ⚡️ Node.js с примерами кода

Справочник web-разработчика с примерами кода - Node.js

https://nodejsdev.ru/doc/setup/

**Comment by messenger91** [comment]
Для Linux пользователей или в WSL достаточно быстро можно установить nvm и довольствоваться возможностью переключением версий ноды
